src/flaskr/app/modules/resources/reset_token/__init__.py

```diff
@@ -0,0 +1,15 @@
+from ...db import Database
+
+class RevokedTokenModel():
+    def __init__(self):
+        self.collection_name = "token_blacklist"
+        self.db = Database()
+
+    def add_token(self, jti):
+        self.db.create(self.collection_name, {
+            "token": jti
+        })
+
+    def is_jti_blacklisted(jti):
+        query = Database.find(self.collection_name, jti)
```

Comment:
.find_one will be faster
Also, this isn't a query, this is a result

Reply:
is the implemented way to find_one()
if you go to Database.find() you going to find:
def find(self, table, selector):
    return self.client.find(table, selector)

if you go to  self.client.find(table, selector) you going to find:
def find(self, table, selector):
  return self.db[table].find_one(selector)